node/src/components/network/event.rs

```diff
@@ -20,7 +20,7 @@ use crate::{
 use core::mem;
 
 const _NETWORK_EVENT_SIZE: usize = mem::size_of::<Event<Message>>();
-const_assert!(_NETWORK_EVENT_SIZE < 89);
+const_assert!(_NETWORK_EVENT_SIZE < 178);
```

Comment:
Why this change?

Reply:
I'm assuming it's down to a libp2p change.  I didn't really want to track it down since we're not focused on libp2p at the moment and I think the original value of `89` was probably chosen based on what the `Event` size was at the time rather than an actual calculated requirement of event size.